lib/tasks_screen/models/task_repository.dart

```diff
@@ -0,0 +1,67 @@
+import 'dart:async';
+
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:lachies_life_planner/shared/config/firebase_config.dart';
+import 'package:lachies_life_planner/tasks_screen/models/task.dart';
+import 'package:lachies_life_planner/tasks_screen/models/task_database_exception.dart';
+
+class TasksRepository {
+  final CollectionReference _taskCollection = getTaskCollection();
```

Comment:
If I only access this in one place, I don't need `getTaskCollection` to be a global method